mito-ai/src/Extensions/ErrorMimeRenderer/ErrorMimeRendererPlugin.tsx

```diff
@@ -64,14 +64,34 @@ class AugmentedStderrRenderer extends Widget implements IRenderMime.IRenderer {
     /**
      * Render the original error message and append the custom prompt.
      */
-    async renderModel(model: IRenderMime.IMimeModel): Promise<void> {    
+    async renderModel(model: IRenderMime.IMimeModel): Promise<void> {
+        // Determine if it's an error or a warning
+        const isErrorMessage = 'application/vnd.jupyter.error' in model.data;
```

Comment:
Would be great to write one small playwright test that just checks when it's a warning the button is not there. 

We write all of our frontend tests in the `mito/tests` folder. You can add a test to the [mitoai.spec.ts](https://github.com/mito-ds/mito/blob/dev/tests/mitoai_ui_tests/mitoai.spec.ts) file. [Here's a test](https://github.com/mito-ds/mito/blob/471766527d7aa64bc87d0b7518bc0072dac52a7f/tests/mitoai_ui_tests/mitoai.spec.ts#L222) that checks if the AI debug button exists that might be a good starting place! 

There's a readme at the top of the `mito/tests` folder with instructions on how to set up the testing locally. There's slightly different commands depending on what part of Mito you are testing, so you can follow the Mac setup commands followed by the Mito AI commands. 

The reason the CI is not passing on this PR is because it's opened as a fork instead of just cloning the repo. Forked PRs don't get access to our CI secrets, as far as I know. Feel free to merge this PR without the test and then open another cloned PR for the frontend test so you can see that it passes.